frontend/src/components/dashboard/Dashboard.tsx

```diff
@@ -54,7 +45,7 @@ const useStyles = makeStyles((theme) => {
       // TODO(design): what color should this actually be -- I
       // don't see it represented in the XD mockup? Ideally
       // it'd be from our emerging design system
-      border: theme.borders.thin,
+      border: `1px solid #e0e0e0`,
```

Comment:
Same here, wonder how did you determine these changes?

Reply:
Same here